src/components/App/App.js

```diff
@@ -0,0 +1,210 @@
+import React, { Component } from 'react';
+import { connect } from "react-redux";
+import { getSymptoms, updateFrequency, getDiagnoses } from '../../actions/index';
+import { PersonIcon, AppHeader, QuestionContainer } from './SmallComponents';
+import './style.css';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    // Set state
+    this.state = {
+      chosenSymptomId: null,
+      rightDisease: null,
+      correctDisease: null
+    }
+  }
+
+  componentDidMount() {
+    this.props.dispatch(getSymptoms());
+    this.props.dispatch(getDiagnoses());
+    this.scrollToBottom();
+  }
+
+  componentDidUpdate() {
+    this.scrollToBottom();
+  }
+
+  scrollToBottom = () => {
+    if (this.el) {
+      this.el.scrollIntoView({ behavior: 'smooth' });
+    }
+  }
+
+  resetPage = (event) => {
+    this.setState({ chosenSymptomId: null,
```

Comment:
Maybe define an initialState and use it to populate your state and refresh it

Reply:
@jacobj - great idea!